src/utils/async-utils.js

```diff
@@ -11,3 +11,9 @@ export const waitForDOMContentLoaded = function() {
     return waitForEvent("DOMContentLoaded", window);
   }
 };
+
+export const sleep = function(milliseconds) {
```

Comment:
Kind of crazy we have an async-utils.js and don't have this heh

Reply:
Not sure if it's any better than `await new Promise(res => setTimeout(res, 5000));`  :shrug:

Reply:
Not sure if it's better than writing `await new Promise(res => setTimeout(res, 5000))` but :shrug: